README.md

```diff
@@ -70,40 +71,36 @@ After enabling the plugin, you need to add the `Dolby.io Subsystem` node to the
 
     If you want to use a client access token, follow these steps:
     
-    1. Right-click the Blueprint to see the `All Possible Actions` window. Tick off `Context Sensitive` to see all available components and actions, and add a `Set Token` action.
+    1. Drag a node off `Dolby.io Subsystem` and select the `Set Token` action.
     <img src="./Images/settoken.png" width="500"/>
     
     2. Connect the `On Token Needed (DolbyIOObserver)` element with the `Set Token` action.
-    
-    3. Connect the `Target` pins of `Set Token` and `Dolby.io Subsystem`.
     <img src="./Images/token.png"/>
     
-    4. Provide your client access token copied from the Dolby.io dashboard to the `Set Token` action.
+    3. Provide your client access token copied from the Dolby.io dashboard to the `Set Token` action.
     
     If you want to use an app key and an app secret, follow these steps:
     
-    1. Right-click the Blueprint to see the `All Possible Actions` window. Tick off `Context Sensitive` to see all available components and actions, and add a `Get Dolby.io Token` component.
+    1. Right-click the empty space in the Blueprint to see the `All Possible Actions` window. Tick off `Context Sensitive` to see all available components and actions and add a `Get Dolby.io Token` function.
     <img src="./Images/gettoken.png" width="500"/>
     
-    2. Right-click the Blueprint and add a `Set Token` action.
+    2. Drag a node off `Dolby.io Subsystem` and select the `Set Token` action.
     
     3. Connect `On Token Needed` with `Get Dolby.io Token`.
     
-    4. Connect `Get Dolby.io Token` with `Set Token`, and connect their `Token` pins.
-    
-    5. Connect the `Target` pins of `Set Token` and `Dolby.io Subsystem`.
+    4. Connect `Get Dolby.io Token` with `Set Token` and connect their `Token` pins.
```

Comment:
Please reword to "Connect the Token Obtained exec pin from Get Dolby.io Token to Set Token and connect their Token pins".